rustls/src/msgs/macros.rs

```diff
@@ -5,18 +5,18 @@ macro_rules! enum_builder {
         #[repr($uint:ty)]
         $enum_vis:vis enum $enum_name:ident
         {
-          $( $enum_var:ident => $enum_val:literal),* $(,)?
+          $( $enum_var_debug:ident => $enum_val_debug:literal),* $(,)?
           $( !Debug:
-            $( $enum_var_nd:ident => $enum_val_nd:literal),* $(,)?
+            $( $enum_var_no_debug:ident => $enum_val_no_debug:literal),* $(,)?
```

Comment:
I am thinking this might read a little better (this does not need any special ignore patterns for typos):

```rs
        $enum_vis:vis enum $enum_name:ident
        {
          $( $enum_var_with_fmt:ident => $enum_val_with_fmt:literal),* $(,)?
          $( !Debug:
            $( $enum_var_no_fmt:ident => $enum_val_no_fmt:literal),* $(,)?
          )?
```

I am now thinking to defer & move these updates out to a separate PR, with some test improvements I am already working on, and simply ignore "enum_.*_nd" in `typos.toml` for now.